### PR TITLE
Update CG-06-18.md

### PR DESCRIPTION
**Pull request by eqrion**
I'd like to present a proposal for https://github.com/WebAssembly/design/issues/1514.

### CODE REVIEW

**Comment by tlively** [comment]
It looks like this was accidentally put into the "future agenda items" part of the schedule. We already have 45 minutes of this meeting allocated, so @eqrion, we can either shorten this to 15 minutes or bump it to another meeting.

**Comment by eqrion** [comment]
> It looks like this was accidentally put into the "future agenda items" part of the schedule. We already have 45 minutes of this meeting allocated, so @eqrion, we can either shorten this to 15 minutes or bump it to another meeting.

Ah, my bad. We should bump it to another meeting then.

**Comment by tlively** [comment]
Done in #1609.
